docs/DeveloperGuide.md

```diff
@@ -564,6 +564,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: Delete a tag**
+
+**MSS**
+
+1.  User requests to list persons.
+2.  FINDvisor shows a list of persons.
+3.  User requests to delete some tags associated with a specific person in the list.
+4.  FINDvisor deletes the tags associated with the specific person.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. FINDvisor shows an error message.
+
+      Use case resumes at step 2.
+
+* 3b. The given tag is invalid.
+
+    * 3b1. FINDvisor shows an error message.
+
+      Use case resumes at step 2.
+
+* 3c. Some of the given tag is not associated with the person.
+
+    * 3c1. FINDvisor shows an error message indicating the tags not found.
+
+      Use case resumes at step 2.
+
+* 4a. The targeted tag is not associated with the person.
+
+    * 4a1. FINDvisor shows a message.
+
+      Use case ends.
```

Comment:
I think we should not be too specific about implementation for use cases? Refer to #183 as to how you can phrase it?